src/main/java/org/genomicsdb/spark/GenomicsDBInput.java

```diff
@@ -359,10 +356,10 @@ private GenomicsDBPartitionInfo addSplitsIfQuerySpansPartitions(
    * Creates export configuration protobuf object 
    * based on partition, query and existing query file or protobuf
    *
-   * @param queryFileOrPB Existing query json file or protobuf string
+   * @param queryFileOrPB Existing query json file or protobuf 
```

Comment:
Change `protobuf` in javadoc to something more explanatory, e.g. `base64 encoded protobuf byte data`

Reply:
done